tests/integration/models/gpflux/test_predictions.py

```diff
@@ -35,19 +36,28 @@ def test_dgp_model_close_to_actuals(depth: int, keras_float: None) -> None:
     num_inducing = 50
     batch_size = 50
     epochs = 500
+    learning_rate = 0.01
 
     example_data = hartmann_6_dataset(dataset_size)
 
-    model, _ = trieste_deep_gaussian_process(
-        query_points=example_data.query_points,
-        depth=depth,
-        num_inducing=num_inducing,
-        learning_rate=0.01,
-        batch_size=batch_size,
-        epochs=epochs,
-        fix_noise=True,
+    def scheduler(epoch: int, lr: float) -> float:
```

Comment:
I think there's a "reduce on plateau" strategy already implemented in Keras, should we use it instead of that one?

Reply:
ok, we will switch to Keras one after we get some experience with it

Reply:
I think ReduceOnPlateau is @vdutor 's default

Reply:
Ah I was not aware of this, I'll have to have a look!